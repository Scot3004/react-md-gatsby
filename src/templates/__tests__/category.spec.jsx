import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Category from '../category';

it('renders correctly', () => {
  const data = {
    "allMarkdownRemark": {
      "edges": [
        "post_edges"
      ]
    }
  }

  const pathContext = {
    category: "Category Name"
  }

  const tree = shallow(<Category data={data} pathContext={pathContext} />);
  expect(toJson(tree)).toMatchSnapshot();
});
