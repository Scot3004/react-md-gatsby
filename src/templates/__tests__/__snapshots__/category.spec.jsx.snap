// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="category-container"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Posts in category "Category Name" | Sergio Orozco
    </title>
    <link
      href="https://www.secorto.com/categories/Category Name"
      rel="canonical"
    />
  </HelmetWrapper>
  <PostListing
    postEdges={
      Array [
        "post_edges",
      ]
    }
  />
</div>
`;
